src/readme.md

```diff
@@ -0,0 +1,35 @@
+# Steps to deploy the function and APIs
+
+- [Import](https://docs.microsoft.com/en-us/azure/devops/repos/git/import-git-repository?view=azure-devops) this repo to an Azure DevOps Repo
+- Create two [ARM service connections](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/connect-to-azure?view=azure-devops) each scoped to the apim resource group and the fucntion app resource group
+- Create a new [Agent pool](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/pools-queues?view=azure-devops&tabs=yaml%2Cbrowser) with name _apim-cs_
```

Comment:
Should this be self hosted or azure agent pool?

Reply:
If its self-hosted we should document the steps to add the host or link it to the host agent in the scripts.

Reply:
@petemessina - This is targeted at the self-hosted agent. We can change it to use the default pool and avoid the step to create a pool